### ISSUE TITLE
Large performance gap between my trained VAE and the released pretrained model in VecSetX

### ISSUE THREAD

**Issue by muradilkurban** [1zb/3DShape2VecSet#17]
I’m training the VAE from VecSetX but the reconstruction quality is bad. Here’s what I did:

Data: Used the ShapeNet car GLB files → extracted SDF/point clouds with sdf_gen/preprocess/box.py → saved as .npz. Visualized samples, data looks fine.
Model: Used learnable_vec1024x16_dim1024_depth24 (no changes).
Training: Ran vecset/main_ae.py with 2 GPUs + gradient accumulation (8 steps). Trained for ~500 epochs.
Results: Final loss ~0.23, volume IoU ~0.64.
Inference: Used vecset/infer.py at 128/256 resolution. Output looks poor.

Any ideas why the quality is low? Thanks! 😊

<img width="782" height="536" alt="Image" src="https://github.com/user-attachments/assets/f16fb093-52e7-4050-b75a-20723c4db8c5" />
